tests/integration/tests.c

```diff
@@ -422,22 +384,23 @@ static void test_graph_type1(void)
 	ASSERT(pg_brick_pkts_count_get(branch1.vhost, PG_EAST_SIDE));
 
 	/* check the collect1 */
-	for (int i = 0; i < 10; i++) {
-		usleep(100000);
-		pg_brick_poll(branch1.vhost_reader, &count, &error);
-		CHECK_ERROR_ASSERT(error);
-		if (count)
-			break;
-	}
-
+	g_assert(pg_graph_poll(graph1, &error) == 0);
 	CHECK_ERROR_ASSERT(error);
 	ASSERT(pg_brick_pkts_count_get(branch1.collect,
 				       PG_EAST_SIDE));
-	ASSERT(count);
 	/* same with VNI 2 */
 	result_pkts = pg_brick_west_burst_get(branch1.collect, &pkts_mask,
 					      &error);
 	ASSERT(result_pkts);
+	/* split graph */
+	graph2 = pg_graph_split(graph, "graph-2", "print", "vt",
+						"q_west", "q_east", &error);
+	check_error(error);
+	/* merge graph after splitted */
+	g_assert(pg_graph_merge(graph, graph2, &error) == 0);
+	check_error(error);
+	g_assert(pg_graph_poll(graph, &error) == 0);
```

Comment:
well that's cool you test this but:
- that's completely unrelated to the pr
- same as before, you remove the usleep...